tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -1003,6 +1003,24 @@ TYPED_TEST(
 		EXPECT_NEAR(c[i] * b[i], a[i], get_epsilon<TypeParam>());
 }
 
+TYPED_TEST(LinalgBackendEigenNonIntegerTypesTest, SGVector_log)
+{
+	SGVector<TypeParam> a{{1.0, 2.0, 3.0}};
```

Comment:
```suggestion
	SGVector<TypeParam> a{1.0, 2.0, 3.0};
```

Reply:
This is the same line that didn't work last time..

Reply:
No it isn't... the error you showed was with a matrix, this is a vector

Reply:
Ohh I understand now.. I will edit it now and push thanks!